src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [status-im2.common.plus-button.view :as plus-button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]))
+
+(defn user-account-view
+  [{:keys [name balance percentage-value amount customization-color watch-only type]}]
```

Comment:
`watch-only` prop is redundant as there is type `watch-only`, you can create an internal let var if you think it make it easier to read.

```
(defn user-account-view
  [{:keys [name balance percentage-value amount customization-color watch-only type]}]
  (let [watch-only? (= :watch-only type)]
  ...
  
 ```